packages/components-react/paragraph/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@nl-design-system-candidate/paragraph-react",
+  "version": "1.0.0",
+  "description": "Paragraph component",
```

Comment:
Misschien moeten we hier de officiële beschrijving gebruiken van de component? Vertaald dan?

NL:
>Toont een alinea aan tekst.

Google Translate EN (helaas een recursieve definitie):

> Paragraph:
> Displays a paragraph of text.